data/maps/Route3/map.json

```diff
@@ -366,9 +422,19 @@
       "x": 26,
       "y": 9,
       "elevation": 3,
-      "item": "ITEM_ORAN_BERRY",
+      "item": "ITEM_SITRUS_BERRY",
       "flag": "FLAG_HIDDEN_ITEM_ROUTE3_ORAN_BERRY",
```

Comment:
Might be worth renaming the flag here to avoid confusing ourselves in the future.